dashboard/src/components/AppView/AppView.tsx

```diff
@@ -14,12 +14,13 @@ import "./AppView.css";
 import ChartInfo from "./ChartInfo";
 import ServiceTable from "./ServiceTable";
 
-interface IAppViewProps {
+export interface IAppViewProps {
   namespace: string;
   releaseName: string;
   app: hapi.release.Release;
-  error: Error;
-  deleteError: Error;
+  // TODO(miguel) how to make optional props? I tried adding error? but the container complains
```

Comment:
Indeed that's weird. I don't know why the signature `error: Error | undefined` is different from `error?: Error`. It's even more weird because in the container we are instantiating the view props with `error: apps.error` where `apps.error` is an `error?: Error`.

Anyway I think that defining it as `Error | undefined` is a good solution.

Reply:
Thanks for checking!

Reply:
I've run into this before also, not sure why TS doesn't like `error?`, but it is specifically an issue when connecting to the Redux, I believe. Child components called from the root component loaded by the container don't have this issue in my experience.